flow/error_definitions.h

```diff
@@ -165,6 +165,7 @@ ERROR( mapper_bad_range_decriptor, 2032, "\"{...}\" must be the last element of
 ERROR( quick_get_key_values_has_more, 2033, "One of the mapped range queries is too large" )
 ERROR( quick_get_value_miss, 2034, "Found a mapped key that is not served in the same SS" )
 ERROR( quick_get_key_values_miss, 2035, "Found a mapped range that is not served in the same SS" )
+ERROR( bg_no_ryw, 2036, "Blob Granule Read Transactions must be specified as ryw-disabled" )
```

Comment:
A suggestion based on the fact that `bg` could be overloaded. Also adjust some casing.

```suggestion
ERROR( blob_granule_no_ryw, 2036, "Blob granule read transactions must be specified as ryw-disabled" )
```